packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -158,6 +158,38 @@ function getCommonEventData<T extends keyof DocumentEventMap>(eventConfig: UIEve
   };
 }
 
+
+// Keep track of event handlers
+const uiEventHandlers = new Map<UIEventConfig['eventName'], {
```

Comment:
This is good, but now we had duplicate sets keeping track of event handler. It is a minor thing as of now, but could become a source of error if we don't keep these synced. May be move the new map and the two functions to the 
other module, and it already has all its public methods exported bellow. 

https://github.com/facebook/hyperion/blob/main/packages/hyperion-autologging/src/ALInteractableDOMElement.ts#L85

Reply:
Should I export this object,  and just have it used in ALInteractableDOMElement (just needs the map keys)?